rules/windows/builtin/appxdeployment_server/win_appxdeployment_server_uncommon_package_locations.yml

```diff
@@ -21,6 +21,7 @@ detection:
     filter_main_generic:
         Path|contains:
             # Paths can be written using forward slash if the "file://" protocol is used
+            - 'AppData\Local\Microsoft\OneDrive\'
```

Comment:
This is an optional software / path. Please add it to a `filter_optional_onedrive` filter.